modules/schema/src/test/java/org/apache/ignite/internal/schema/RowTest.java

```diff
@@ -421,6 +433,26 @@ else if (type == NativeTypeSpec.STRING) {
                         nonNullVarLenValSize += RowAssembler.utf8EncodedLength((CharSequence)vals[i]);
                     }
                 }
+                else if (type == NativeTypeSpec.NUMBER) {
+                    if (schema.isKeyColumn(i)) {
+                        nonNullVarLenKeyCols++;
+                        nonNullVarLenKeySize += NumericTypeUtils.sizeInBytes((BigInteger)vals[i]);
```

Comment:
Let's move sizeInBytes method to RowAssembler.

Reply:
Done